### PR TITLE
Check if array property exists

### PR DESCRIPTION
**Pull request by ghost**
Pull Request for Issue #18043

### Summary of Changes

Added one line above 183

### CODE REVIEW

**Comment by C-Lodder** [comment]
Could you please add some testing instructions?

File: libraries/src/Component/Router/Rules/MenuRules.php [hunk 1]
```diff
@@ -179,49 +179,49 @@ protected function buildLookup($language = '*')
...
-						}
-
-						if (!isset($this->lookup[$language][$view]))
-						{
-							$this->lookup[$language][$view] = array();
-						}
-
-						// If menuitem has no key set, we assume 0.
-						if (!isset($item->query[$views[$view]->key]))
+					if(!empty($view))
```
Comment (Quy) [inline libraries/src/Component/Router/Rules/MenuRules.php:182]: Add a space after `if`.
Missing `{` and `}` for the newly added `if`.

**Comment by Quy** [comment]
Duplicate #17323 @coderdynamic Please test.

**Comment by ghost** [comment]
@Quy i closed #17323, should this also be closed as duplicate PR?

**Comment by Quy** [comment]
Yes, please.

**Comment by ghost** [comment]
closed as duplicate Pull Request.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18123">issues.joomla.org/tracker/joomla-cms/18123</a>.</sub>

**Pull request closed without merging**